app/components/application_component.rb

```diff
@@ -9,4 +9,12 @@ class ApplicationComponent < ViewComponent::Base
   def merge_classes(*)
     ComponentSupport::CssClasses.merge(*)
   end
+
+  # Merge data-actions together.
+  #
+  # @param args [Array<String>, String] The actions to merge (array, classes, space separated classes).
+  # @return [String] The merged classes.
+  def merge_actions(*)
+    ComponentSupport::CssClasses.merge(*)
+  end
```

Comment:
Identical implementation to `#merge_classes`?

Reply:
Yes. Even though the implementation is currently the same, having separate method names is clearer.

Reply:
@justinlittman No argument there. What I'm concluding: this change reveals that `Component::CssClasses.merge` has little (nothing?) to do with CSS classes. I suspect we named it `CssClasses` because of the output it was receiving, which now includes both CSS classes and actions (and may include more as we go along). The method is more about unique string joins, apparently. We could consider renaming the containing class?